src/components/HotjarSurvey/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react';
+import { useGlobalState, useStepInfo, useAssessmentStepConfig } from 'hooks/app';
+import { stepNames, stepStates } from 'constants';
+
+function ShowSurvey() {
```

Comment:
please use camel case. This is a regular function. Only react function should be capitalize.

Reply:
I changed it after lint check error